pkg/catalog/outbound_traffic_policies.go

```diff
@@ -292,6 +301,10 @@ func (mc *MeshCatalog) ListMeshServicesForIdentity(identity identity.ServiceIden
 	splitPolicy := mc.meshSpec.ListTrafficSplits()
 
 	for upstreamSvc := range dstServicesSet {
+		// Traffic Splits aren't yet supported for non-local services.
+		if !upstreamSvc.Local() {
```

Comment:
I am not sure we would need this. If the service mentioned in the traffic split isn't present on the local cluster but there are remote clusters that have the service ( we could get this from the MultiClusterService) then the traffic should be split to all those remote services in the round robin manner with the max weight being the weight specific on the traffic split.

Example: if bookstore-v1 is only on cluster x and cluster y and the weight is 50 then we would split the traffic to 25 on each of the clusters

Reply:
That's actually goes against the intent of the design, and is counterintuitive to users. If I reference `bookstore-v1` in namespace `bookstore`, then that gets resolved to `bookstore-v1.bookstore.svc.cluster.local`. It should never be the case that `.cluster.local` sends traffic to the non local cluster.

Reply:
post our offline discussion we will let go ahead with your proposal